frontend/packages/dev-console/src/components/add/DeveloperFeaturesGettingStartedCard.tsx

```diff
@@ -2,7 +2,8 @@ import * as React from 'react';
 import { FlagIcon } from '@patternfly/react-icons';
 import { useTranslation } from 'react-i18next';
 import * as semver from 'semver';
-import { ALL_NAMESPACES_KEY, useActiveNamespace, useOpenShiftVersion } from '@console/shared/src';
+import { useActiveNamespace } from '@console/dynamic-plugin-sdk';
+import { ALL_NAMESPACES_KEY, useOpenShiftVersion } from '@console/shared/src';
```

Comment:
We discussed at one point returning `undefined` instead of `ALL_NAMESPACES_KEY`. However there's also the case of when no namespace is set, we should use the default namespace.

So either expose `ALL_NAMESPACES_KEY` as valid API or we change the API now.

cc @spadgett @vojtechszocs

Reply:
It looks like there are a few cases where we treat not set and `ALL_NAMESPACES_KEY` differently that we'd need to handle if we change the API. For instance:

https://github.com/spadgett/console/blob/a406d765a7b7930f8a129d185c260495488cf96e/frontend/public/components/RBAC/role.jsx#L254

Reply:
If we keep `ALL_NAMESPACES_KEY` then we need to expose this as an API constant.